src/library_glfw.js

```diff
@@ -1707,7 +1741,21 @@ var LibraryGLFW = {
 
   glfwGetClipboardString: function(win) {},
 
-  glfwMakeContextCurrent: function(winid) {},
+  glfwMakeContextCurrent__deps: ['emscripten_webgl_make_context_current'],
+
+  glfwMakeContextCurrent: function(winid) {
+    if(winid !== null && GLFW.windows !== null) {
+      for (i = 0; i < GLFW.windows.length && GLFW.windows[i].id != winid; i++) {
+        // no-op
+      }
+      if(i < GLFW.windows.length) {
+        Module['ctx'] = Module['ctx' + (winid - 1)];
+      }
```

Comment:
Can you make this into a single loop:

```
for (var i in GLFW.windows) {
  if (GLFW.windows[i].id == winid) {
     Module['ctx'] = Module['ctx' + (winid - 1)];
     break;
  }
}
```

Reply:
alright

Reply:
Done: 312dfb40cc64c0b9f5770ff45b2f01cfc7fea170